README.md

```diff
@@ -1,56 +1,58 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Cinema
 
-> Description the project.
+_We will add the description later_
 
+# Testing Jest
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+> Testing certain functions and methods using Jest
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- HTML, CSS, JavaScript
+- Webpack
+- Jest
+- NPM
+- Node
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+In order to obtain a local copy of this repository run:
 
-
-To get a local copy up and running follow these simple example steps.
+- `git clone https://github.com/BucurEva87/cinema.git`
+- `cd cinema`
 
 ### Prerequisites
 
-### Setup
+- Install Node and Node Package Manager (NPM)
+- Install all the project dependencies running `npm i`
 
-### Install
+### Running tests
 
-### Usage
+- You can run test suites on this project by running `npm test`
 
-### Run tests
+## Live Demo
 
-### Deployment
+[Live Demo Link](https://bucureva87.github.io/cinema/dist/)
```

Comment:
- [ ] [OPTIONAL] It would be helpful if you deploy your project using `Netlify` or `Heroku`, as the `GitHub Pages` deployment link you posted is not working.

![image](https://user-images.githubusercontent.com/63915024/188992861-21d27b83-65f2-4c23-8624-29c2dce89644.png)